src/P3_processes.jl

```diff
@@ -76,31 +76,23 @@ function ice_melt(
     # Note: process not dependent on `F_liq`
     # (we want ice core shape params)
     # Get constants
-    (; ν_air, D_vapor, K_therm) = aps
+    (; K_therm) = aps
     L_f = TD.latent_heat_fusion(tps, Tₐ)
-    N_sc = ν_air / D_vapor
 
-    state = get_state(dist)
-    params = get_parameters(dist)
+    (; L, N, state) = dist
+    (; T_freeze) = state.params
 
-    # Ice particle terminal velocity
-    v(D) = ice_particle_terminal_velocity(state, D, Chen2022, ρₐ)
-    # Reynolds number
-    N_Re(D) = D * v(D) / ν_air
-    # Ventillation factor
-    (; vent_a, vent_b) = params.vent
-    F_v(D) = vent_a + vent_b * N_sc^FT(1 / 3) * N_Re(D)^FT(1 / 2)
+    F_v = ventilation_factor(state, Chen2022, ρₐ, aps)
```

Comment:
Maybe `P3_state` instead of `state`? I still find it confusing given that we use `state` for the whole model state in Atmos

Reply:
ok, I'll revert to p3_state.